.github/workflows/psammead-manual-talos.yml

```diff
@@ -1,10 +1,10 @@
-name: Psammead CD - Manual Talos
+name: Manual Talos - ${{ github.event.inputs.psammeadPackages }}
```

Comment:
I am not sure if the inputs will work in this title. As this is just the main title of the workflow.